code/datums/diseases/advance/symptoms/fire.dm

```diff
@@ -33,15 +33,25 @@ Bonus
 			if(3)
 				M << "<span class='warning'>[pick("You feel hot.", "You hear a crackling noise.", "You smell smoke.")]</span>"
 			if(4)
-				M.adjust_fire_stacks(5)
+				Firestacks1(M, A)
 				M.IgniteMob()
 				M << "<span class='userdanger'>Your skin bursts into flames!</span>"
-				M.adjustFireLoss(5)
 				M.emote("scream")
 			if(5)
-				M.adjust_fire_stacks(10)
+				Firestacks2(M, A)
 				M.IgniteMob()
 				M << "<span class='userdanger'>Your skin erupts into an inferno!</span>"
-				M.adjustFireLoss(10)
 				M.emote("scream")
-	return
\ No newline at end of file
+	return
+
+/datum/symptom/fire/proc/Firestacks1(mob/living/M, datum/disease/advance/A)
```

Comment:
Ah, so I should have then with better descriptive names i.e. stage4firestack and stage5firestack or make two functions that use get_stacks_stage_4 and get_stacks_stage_5 instead?